samples/consoleapp/getting-started-for-dotnet-real-time/PBIRealTimeStreaming/PBIRealTimeStreaming/Program.cs

```diff
@@ -0,0 +1,99 @@
+﻿//Copyright Microsoft 2016
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;
+
+namespace PBIRealTimeStreaming
+{
+
+    // Sample app demonstrating how to use the Power BI REST APIs for streaming data
+    // Pushes current datetime and a random integer to the REST API every second
+    // Full documentation: https://powerbi.microsoft.com/documentation/powerbi-service-real-time-streaming/
+
+    // To run this sample:
+    // 1. Go to app.powerbi.com
+    // 2. Go to streaming data management page by via new dashboard > Add tile > Custom Streaming Data > manage data
+    // 3. Click "Add streaming dataset"
+    // 4. Select API, then Next, and give your streaming dataset a name
+    // 5. Add a field with name "ts", type DateTime
+    // 6. Add a field with name "value", type Number
+    // 7. Click "Create"
+    // 8. Copy the "push URL" and paste it as the value of "realTimePushURL" below
+
+    class Program
+    {
+
+        // Paste your push URL below
+        private static string realTimePushURL = "** paste your push URL here **";
```

Comment:
I would add an example of what the URL is or how to construct the proper URL.  ex) `https://api.powerbi.com/...`